CMakeLists.txt

```diff
@@ -405,6 +405,15 @@ add_custom_target(check-integration
     USES_TERMINAL
     VERBATIM)
 add_dependencies(check-integration test-gravity)
+add_custom_target(check-themes
+    tests/themes/run.sh
+    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+    COMMENT "Testing themes"
+    USES_TERMINAL
+    DEPENDS generated_icons generate_awesomerc ${PROJECT_AWE_NAME}
+    VERBATIM)
+add_dependencies(check-integration check-themes)
```

Comment:
If `check-themes` fails, the other integration tests are not being run.
That applies to `test-gravity` also.
Should we rather only run them explicitly (on Travis)?
There's no point really to run `check-themes` locally most of the time.